modules/renter/hostdb/consts.go

```diff
@@ -34,6 +34,9 @@ const (
 	// scans start getting compressed.
 	minScans = 12
 
+	// halftimeUpDownTime is the halftime used to decay the host up and downtime
+	halftimeUpDownTime = 30 * 24 * time.Hour
```

Comment:
I think the word that you want here is halflife. I'd probably call this `uptimeHalflife`